packages/owlbot-bootstrapper/common-container/test/pre-process.test.ts

```diff
@@ -126,4 +127,28 @@ describe('pre processing', async () => {
     await assert.rejects(() => preProcess(argv));
     scope.done();
   });
+
+  it('does not open an issue if test = true', async () => {
+    argv = {
+      projectId: 'myprojects',
+      apiId: 'google.cloud.kms.v1',
+      language: 'nodejs',
+      installationId: '12345',
+      repoToClone: 'git@github.com/googleapis/nodejs-kms.git',
+      monoRepoPath: 'MONO_REPO_PATH',
+      monoRepoName: 'nodejs-kms',
+      monoRepoOrg: 'googleapis',
+      monoRepoDir: 'MONO_REPO_DIR',
+      serviceConfigPath: 'SERVICE_CONFIG_PATH',
+      interContainerVarsPath: 'INTER_CONTAINER_VARS_PATH',
+      buildId: '1234',
+      skipIssueOnFailure: 'true',
+    };
+
+    const octokit = new Octokit({auth: 'abc1234'});
+    authenticateOctokitStub.returns(octokit);
+    cloneRepoAndOpenBranchStub.rejects();
+
+    await assert.rejects(() => preProcess(argv));
```

Comment:
We generally want to assert the type of error. This could pass for a method call on an undefined object too.